enterprise/app/filter/filter.tsx

```diff
@@ -385,13 +450,56 @@ export default class FilterComponent extends React.Component<FilterProps, State>
                     />
                   </div>
                   <div className="option-group-input">
-                    <FilledButton onClick={this.handleApplyClicked.bind(this)}>Apply</FilledButton>
+                    <FilledButton onClick={this.handleFilterApplyClicked.bind(this)}>Apply</FilledButton>
                   </div>
                 </div>
               </form>
             )}
           </Popup>
         </div>
+        <div className="popup-wrapper">
+          <OutlinedButton className="sort-button icon-text-button" onClick={this.onOpenSortMenu.bind(this)}>
```

Comment:
nit: in the case where the button text is empty, it looks a bit too wide. It would be nice to have it look nice and square like the filter icon. When the text is empty, you can use the class `icon-button` instead of `icon-text-button` and remove the empty `<span>` element.

Reply:
Good call, fixed.